docs/day2/02_fastqc.md

```diff
@@ -0,0 +1,329 @@
+# 2.2 Samplesheets, operators, and groovy  
+
+!!! note "Learning objectives"
+
+    1. Implement a process with a tuple input
+    2. Understand why samplesheets should be used to read in data
+    3. Create a channel with operators and Groovy
+
+In this step we will implement `01_fastqc.sh` as a process called `FASTQC` and
+use operators and Groovy to create a channel.  
+
+![](img/2.excalidraw.png)
```

Comment:
Same as previous process, I think we need a diagram with a big more context, can highlight this section.